Corona-Warn-App/src/main/java/de/rki/coronawarnapp/srs/core/repository/SrsSubmissionRepository.kt

```diff
@@ -164,8 +164,7 @@ class SrsSubmissionRepository @Inject constructor(
                 val errorCode = e.type.toSrsErrorType()
                 SrsSubmissionException(
                     errorCode = errorCode,
-                    errorArgs = errorCode.errorArgs(appConfig.selfReportSubmission),
-                    cause = e
```

Comment:
Passing the cause will lead to `SrsSubmissionException` parcelization errors. We aren't using the cause to display error dialogs anyway.

Reply:
would `@IgnoredOnParcel` prevent the crash?

Reply:
I'll give it a try

Reply:
Update: nope, still crashing with `@IgnoredOnParcel`